mettle/worker/worker.go

```diff
@@ -513,6 +517,13 @@ func (w *worker) prepareDir(action *pb.Action, command *pb.Command) *rpcstatus.S
 		if grpcstatus.Code(err) == codes.NotFound {
 			log.Notice("Incrementing blobNotFoundErrors")
 			blobNotFoundErrors.Inc()
+			if w.promGatewayURL != "" {
+				if err := push.New(
+					w.promGatewayURL, "blob_not_found_errors_total",
+				).Collector(blobNotFoundErrors).Format(expfmt.FmtText).Push(); err != nil {
```

Comment:
Rather than pushing metrics when we increment this one individually, I think we should push continuously in the background if enabled (say every few minutes or something).
Without that, if a worker never hits this error, it will never push any of the other metrics.

Reply:
Whoops, yes